### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Jonathan Thiebaud

### DISCUSSION THREAD

**Discussion by JONATHAN-THIEBAUD** [heig-vd-dai-course/heig-vd-dai-course#287] [Show and tell]
https://github.com/JONATHAN-THIEBAUD/NeyZ

**Comment by ludelafo** [comment]
Thank you! If you want your profile to be visible when someone accesses your GitHub account (https://github.com/JONATHAN-THIEBAUD), you should name your repository `JONATHAN-THIEBAUD`. :)
